hawkmoth/parser.py

```diff
@@ -156,6 +156,14 @@ def _comment_extract(tu):
                                  CursorKind.MACRO_INSTANTIATION]:
             continue
 
+        # Cursors that are 1) never documented themselves, and 2) not allowed
+        # between the comment and the actual cursor being documented.
+        if token_cursor.kind in [CursorKind.UNEXPOSED_DECL]:
+            if current_comment and docstring.Docstring.is_doc(current_comment.spelling):
```

Comment:
Side note, `current_comment and docstring.Docstring.is_doc(current_comment.spelling)` could use a helper, it's already repeated quite a bit. Does *not* need to be part of this series.